packages/react-codemirror/src/lang-cypher/signatureHelp.ts

```diff
@@ -45,16 +45,7 @@ function getSignatureHelpTooltip(
       ) {
         const signature = signatures[activeSignature];
         const parameters = signature.parameters;
-        let doc = signature.documentation.toString();
-
-        if (
-          activeParameter >= 0 &&
-          activeParameter <
-            (signatures[activeSignature].parameters?.length ?? 0)
-        ) {
-          doc =
-            parameters[activeParameter].documentation.toString() + '\n\n' + doc;
-        }
```

Comment:
This was a safeguard in case we return an active parameter of -1, which if I recall correctly it could happen, so please don't remove

Reply:
Ah, that was not my intention, I'll add back a safe guard :+1:

Reply:
Hmm, wait looking into it now, it's still there just moved